png_benchmark.py

```diff
@@ -0,0 +1,152 @@
+                gc.enable()
+
+                # Save singular test results
+                time_ns = end - start
+                size_B = buffer.getbuffer().nbytes
+                compression_ratio = size_B / os.path.getsize(pathlib.Path(INPUT_PATH, f"{frame_index}.png")) * 100
+
+                if time_ns > max_time_ns:
+                    max_time_ns = time_ns
+                elif time_ns < min_time_ns:
+                    min_time_ns = time_ns
+                
+                if size_B > max_size_B:
+                    max_size_B = size_B
+                elif size_B < min_size_B:
+                    min_size_B = size_B
+                
+                if compression_ratio > max_compression_ratio:
+                    max_compression_ratio = compression_ratio
+                elif compression_ratio < min_compression_ratio:
+                    min_compression_ratio = compression_ratio
+                
+                total_time_ns += time_ns
+                total_size_B += size_B
+                total_compression_ratio += compression_ratio
+                test_result = {
+                    "time_ns": time_ns,
+                    "size_B": size_B,
+                    "size_ratio_compressed_to_original_%": compression_ratio
+                }
+                results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"]["frame_data"].append(test_result)
+
+                # Save one image (this one has 2 landing pads in it) for reference
+                if frame_index == FRAME_TO_SAVE:
+                    img.save(pathlib.Path(OUTPUT_PATH, f"ct{compress_type}_cl{compress_level}.png"), "PNG", compress_level=compress_level, compress_type=compress_type)
+            
+            # Save average test results
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MAX_TIME_MS] = max_time_ns / 1e6
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MIN_TIME_MS] = min_time_ns / 1e6
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][AVG_TIME_MS] = total_time_ns / FRAME_COUNT / 1e6
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MAX_SIZE_B] = max_size_B
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MIN_SIZE_B] = min_size_B
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][AVG_SIZE_B] = total_size_B / FRAME_COUNT
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MAX_RATIO_COMPRESSED_TO_ORIGINAL] = max_compression_ratio
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MIN_RATIO_COMPRESSED_TO_ORIGINAL] = min_compression_ratio
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][AVG_RATIO_COMPRESSED_TO_ORIGINAL] = total_compression_ratio / FRAME_COUNT
```

Comment:
Line too long, maximum is 100 characters. Use intermediate string variables and/or intermediate dictionary.